dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/config/StoreConfiguration.java

```diff
@@ -41,6 +43,10 @@ public class StoreConfiguration {
     @Bean
     public StorageOperate storageOperate() {
         switch (PropertyUtils.getString(RESOURCE_STORAGE_TYPE)) {
+            case STORAGE_OSS:
+                OssOperator ossOperator = OssOperator.getInstance();
+                ossOperator.init();
```

Comment:
Since this is a singleton, why you don't put the `init` method into constructor? Can the init method be executed multiple times?

Reply:
This is also for code-decoupling and testing purpose. If we put this in the constructor, `oss client` will get instantiated in the constructor. As a result, we lose control of the instantiation of the external class and could not inject the corresponding mock during testing.